gamedb.yaml

```diff
@@ -1403,6 +1403,12 @@
   poster: https://cdn2.steamgriddb.com/grid/db942ab6ce5deb8d2dd0a50d401e40cb.jpg
   id: 284390
 
+# SteamDeck=0 to allow resolutions above 1080P
+- name: Last Epoch
+  platform: steam
+  id:  	899770
```

Comment:
The tests are failing because there is a syntax error due to a tab on this line, after the ':'.
Please replace with a space.